src/checkers/ast.rs

```diff
@@ -4472,7 +4472,7 @@ impl<'a> Checker<'a> {
                     let multiple = unused_imports.len() > 1;
                     for (full_name, range) in unused_imports {
                         let mut diagnostic = Diagnostic::new(
-                            violations::UnusedImport(full_name.to_string(), ignore_init, multiple),
+                            violations::UnusedImport{name:full_name.to_string(), ignore_init, multiple},
```

Comment:
I'll go ahead and do it in this case, but for the future, you'll need to run `cargo +nightly fmt` to get the code in good shape for CI.